src/Workspaces/CSharp/Portable/Extensions/CastExpressionSyntaxExtensions.cs

```diff
@@ -144,7 +144,7 @@ private static bool UserDefinedConversionIsAllowed(ExpressionSyntax expression,
             SemanticModel semanticModel,
             CancellationToken cancellationToken)
         {
-            // Special case: When a null literal is cast and passed as the single argument to a params parameter,
+            // When a casted parameter is passed as the single argument to a params parameter,
```

Comment:
💭 I actually don't know if this should be **cast** or **casted**. But either way, value seems to be more appropriate than parameter so I would suggest the following:

> casted ~~parameter~~ **value**